### PR TITLE
APM-341627

### PR DESCRIPTION
**Pull request by kamiljsokolowski**
This change will:

1. Disable executing the pipeline on PR creation and updates.
2. Disable executing the pipeline on PRs coming from forks.

### CODE REVIEW

**Review by karolkieglerski — CHANGES_REQUESTED** [review]
Please start from APM-310059-move-gcp-assets-to-ef not from master

**Comment by kamiljsokolowski** [comment]
> Please start from APM-310059-move-gcp-assets-to-ef not from master

Done.

After the merge, I needed to add a condition to release to GitHub only on the release tag.

File: .travis.yml [hunk 1]
```diff
@@ -50,6 +51,7 @@ jobs:
 
   - stage: github-deploy
     name: GitHub deploy
+    if: tag =~ /^release-.*
```
Comment (karolkieglerski) [inline .travis.yml:54]: why only on release?

File: .travis.yml [hunk 2]
```diff
@@ -106,7 +108,7 @@ jobs:
     language: python
     python:
       - "3.8"
-    if: (branch = master OR type = pull_request) AND fork = false
+    if: branch = master OR tag =~ /^release.*$/
```
Comment (karolkieglerski) [inline .travis.yml:111]: we would like to run that tests also on feature branches (function tests also)

File: .travis.yml (hunk 1 above)
Comment (kamiljsokolowski) [reply .travis.yml:54]: Missed it.

File: .travis.yml (hunk 2 above)
Comment (kamiljsokolowski) [reply .travis.yml]: Done.

**Review by dtwwolinski — APPROVED** [review]

**Pull request merged**